nselib/data/http-sql-errors.lst

```diff
@@ -39,13 +46,14 @@ MySQL ODBC
 ODBC DB2
 ODBC Driver
 ODBC Error
-ODBC Microsoft Access
+ODBC Microsoft Access Driver
 ODBC Oracle
 ODBC SQL
 ODBC SQL Server
 OLE/DB provider returned message
 ORA%-0
 ORA%-1
+ORA-01756
```

Comment:
Need to escape the `-` here, but isn't this a subset of the earlier `ORA%-0` match?